IFC4x3/Properties/t/TransformationRatio_0z_ZR$vVrEivWXuanAlBNq/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="TransformationRatio_0z_ZRvVrEivWXuanAlBNq" Name="TransformationRatio" UniqueId="3dfa36ff-e5fd-4eb3-9821-e24c4abcb5f4" PrimaryDataType="IfcReal" />
```

Comment:
Would PrimaryDataType be better suited to be one of the `Ifc...RatioMeasure` types?

Reply:
Agree. Commit : b5e658cad21e4944423ed95d945268c722299c68